stacks-signer/release-process.md

```diff
@@ -63,10 +63,11 @@ The timing of the next Stacking cycle can be found [here](https://stx.eco/dao/to
    - Add cherry-picked commits to the `feat/signer-X.Y.Z.A.n.x-pr_number` branch
    - Merge `feat/signer-X.Y.Z.A.n.x-pr_number` into `release/signer-X.Y.Z.A.n.x`.
 
-4. Open a PR to update the [CHANGELOG](./CHANGELOG.md) file in the `release/signer-X.Y.Z.A.n.x` branch.
+4. Open a PR to update the [CHANGELOG](./CHANGELOG.md) and [libsigner/src/libsigner.rs](../libsigner/src/libsigner.rs) files in the `release/signer-X.Y.Z.A.n.x` branch.
```

Comment:
What needs to change in libsigner/src/libsigner.rs?

Reply:
Sorry that was from a previous iteration - I've removed